### ISSUE TITLE
Not able to see AWS attributes in the list  as attached

### ISSUE THREAD

**Issue by Harish1027** [Netflix/security_monkey#778]
![image](https://user-images.githubusercontent.com/30609587/28805591-dbaa4d80-7688-11e7-833b-e990be1a44d8.png)
![untitled](https://user-images.githubusercontent.com/30609587/28805600-eac385fc-7688-11e7-8975-cfe51b5628eb.png)

Need help to monitor my AWS account

**Comment by mikegrima** [comment]
Hello @Harish1027 :

Did you add any accounts to Security Monkey?  You will need to restart the scheduler after adding new accounts.  Please read this section of the user guide, namely the section about restarting the scheduler: https://github.com/Netflix/security_monkey/blob/develop/docs/userguide.md#adding-an-account-in-the-web-ui

**Comment by Harish1027** [comment]
Hi Mikegrima,

Thank for reply.

I have added the account to security monkey and restarted the scheduler but still the same.

I can see the account in the setting tab but not able to see the AWS attribute to filter.

Is there any watcher or plugins need to install or start  services like 

" monkey run_api_server".

I can see the issue at login screen itself I didnt get any option to register as shown in user guide 


![image](https://user-images.githubusercontent.com/30609587/28839635-e57af768-7711-11e7-8361-07411e64d149.png)

I have followed the quick start guide fro setup https://github.com/Netflix/security_monkey/blob/develop/docs/quickstart.md.
